terraform/Makefile

```
@@ -1,11 +1,13 @@
 TARGET_OS_ARCH:=$(shell go env GOOS)_$(shell go env GOARCH)
+UNSUPPORTED_PROVIDERS:=$(shell echo $SKIP_TERRAFORM_PROVIDERS)
 
 TFSUBDIRS:=	$(foreach DIR,$(shell find providers -maxdepth 1 -mindepth 1 -type d),$(subst providers/,,$(DIR)))
+SUPPORTEDTFSUBDIRS:= $(patsubst $(UNSUPPORTED_PROVIDERS),,$(TFSUBDIRS))
```

```suggestion
SUPPORTEDTFSUBDIRS:= $(filter-out $(UNSUPPORTED_PROVIDERS),$(TFSUBDIRS))
```

using `patsubst` assumes that there is only ever 1 unsupported provider I think

---

patsubst does replace multiple occurrences of a pattern. In this case, both work. I have replaced it with filter-out.